### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by TonyFwin** [guilty-as/hubspot-connector#2]
Error when editing settings even after updating to version 1.0.3. So I updated from version 1.0.2 to version 1.0.3 and added in my Hubspot API Key successfully, but after trying to play around with on with pulling blogs in the templates I get an internal server error when using the limit blog posts example. On top of that I'm experiencing the CP internal server error again.

**Comment by HelgeSverre** [comment]
Which PHP version are you using?

**Comment by TonyFwin** [comment]
7.1.26

**Comment by HelgeSverre** [comment]
Did you experience cp errors only after getting an error in the template,
does this error go away when removing the code from the template?

**Comment by TonyFwin** [comment]
Yes, I experienced the CP errors after using the template code and the CP error continues even after the template code is removed.

**Comment by HelgeSverre** [comment]
What craft version are you using?
Are you using projectConfig in your project?
Does the error go away if you remove hubspot connector from yoyr composer
file and run composer update?
